bears/vcs/git/CISkipInspectBear.py

```diff
@@ -0,0 +1,67 @@
+import re
+
+from bears.vcs.git.GitCommitMetadataBear import GitCommitMetadataBear
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+from coalib.settings.Setting import typed_list
+
+
+class CISkipInspectBear(GlobalBear):
+    LANGUAGES = {'Git'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    BEAR_DEPS = {GitCommitMetadataBear}
+
+    SKIP_CI_REGEX = r'\[ci skip\]|\[skip ci\]'
+
+    def run(self, dependency_results,
+            filename_patterns: typed_list(str),
+            appveyor_ci: bool = False,
+            **kwargs):
+        """
+        Most CI allow commits to skip CI build by including sequences
+        like [skip ci] or [ci skip] anywhere in the commit message.
+        AppVeyor CI supports [skip appveyor] in addition to the above
+        but only in the commit title.
+        This bear checks the HEAD commit to see if it disables CI build
+        and return result accordingly. Supported CI include AppVeyor,
+        Bitrise, Circle CI, GitLab CI, Scrutinizer, Semaphore,
+        Shippable, Travis CI and wercker.
+
+        :param filename_patterns:   A list of regex to identify files
```

Comment:
Each .coafile section already has a `files` setting (and `ignore`).  These are already populated for you with real filenames.  Use it for the list of files.

Reply:
@jayvdb Could you please give me an example bear which uses these settings. I am not able to understand how to do it. Also, I think in section like `[all]`, we can have files of type `tests/**/*.py` in `files` setting which are to be ignored. How should I manage that?

Reply:
access `self.section.files`

Reply:
There is nothing like `section.files` in https://github.com/coala/coala/blob/master/coalib/settings/Section.py#L124
Could you please suggest any other solution @jayvdb

Reply:
I think what @jayvdb meant is `self.section['files']`, see https://github.com/refeed/coala-bears/blob/master/bears/c_languages/codeclone_detection/ClangFunctionDifferenceBear.py#L144